extension/stats/index.js

```diff
@@ -1,16 +1,34 @@
 let history = JSON.parse(localStorage.getItem("history"));
-let colorRange = "#F9BB2D";
+let chartColor = "#F9BB2D";
 let weeks = { "Sun": 0, "Mon": 0, "Tue": 0, "Wed": 0, "Thu": 0, "Fri": 0, "Sat": 0 };
 let dates = num(1, 31);
 let hours = num(0, 23);
-let stats = [{ name: "docsss", value: 0, color: "#f00" }, { name: "docss", value: 0, color: "#ff00af" }, { name: "docs", value: 0, color: "#b600ff" }, { name: "%", value: 0, color: "#3400ff" }, { name: ">", value: 0, color: "#0944ff" }, { name: "@", value: 0, color: "#00f2ff" }, { name: "#", value: 0, color: "#00ff2d" }, { name: "text", value: 0, color: "#ffa600" }]
+function num(start, end, initial = 0) {
```

Comment:
We should give it a more intuitive function name, such as `makeNumericKeyObject()`?